### ISSUE TITLE
User -> Access list does not group Roles correctly

### ISSUE THREAD

**Issue by chrismeyersfsu** [ansible/awx#7518]
![image](https://user-images.githubusercontent.com/722880/86280548-015e1980-bbaa-11ea-9655-173c2c604d9a.png)

Roles for the same related resource should live on the same line. From the above screenshot I expected only 2 rows, not 4. Each row should have roles `admin, read`

**Comment by trahman73** [comment]
Expected grouping style: https://tower-mockups.testing.ansible.com/patternfly/orgs/orgs-users/

**Comment by AlexSCorey** [comment]
The way our pagination works currently we can't group the roles in this way because the number of items the api sends will not match the number of rows on the list.  Consolidating like roles/resources will interfere with the pagination and sorting

**Comment by unlikelyzero** [comment]
During `needs_test` verification, please add testcase (or stub)

**Comment by mabashian** [comment]
I agree this would be nice but as @AlexSCorey points out I don't think this is possible given the current structure of the API response on this endpoint.

**Comment by mabashian** [comment]
Lets look and see how we used to do this in the old UI and confirm whether or not this is possible.
